src/libs/actions/Policy.js

```diff
@@ -247,10 +248,15 @@ function loadFullPolicy(policyID) {
                 return;
             }
 
-            Onyx.set(`${ONYXKEYS.COLLECTION.POLICY}${policy.id}`, {
-                ...allPolicies[`${ONYXKEYS.COLLECTION.POLICY}${policy.id}`],
-                ...getSimplifiedPolicyObject(policy, true),
-            });
+            const currentPolicy = _.clone(allPolicies[`${ONYXKEYS.COLLECTION.POLICY}${policyID}`]);
+            const simplifiedPolicyObject = getSimplifiedPolicyObject(policy, true);
+
```

Comment:
Unnecessary blank line

Reply:
Okay

Reply:
? Are you going to remove it?

Reply:
Yes. my plan is to remove it. Why? Is there any change?

Reply:
I am sorry. I have made change in my local file, it is for me so I am not missing which one is already modified and which one is not yet modified.